src/braket/jobs/quantum_job.py

```diff
@@ -162,12 +166,12 @@ def result(
 
 
         Returns:
-            Dict[str, Any]: Dict specifying the hybrid job results.
+            dict[str, Any]: Dict specifying the hybrid job results.
 
         Raises:
             RuntimeError: if hybrid job is in a FAILED or CANCELLED state.
             TimeoutError: if hybrid job execution exceeds the polling timeout period.
-        """
+        """  # noqa: D402
```

Comment:
Why is this needed?

Reply:
`save_job_result()` in the description violates the rule.